package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@psychedelic/dab-js",
-  "version": "1.0.2",
+  "version": "1.0.4",
```

Comment:
Was that bumped manually? Not a big deal otherwise:

```suggestion
  "version": "1.0.3",
```

Reply:
Yeah, had an issue with yarn caching the old version after I pushed an update, so just had to bump up a second time

Reply:
@ozwaldorf Ok np! Another way you might trick it is to use suffix -beta1, -alphaxxx, etc.

Reply:
@heldrida Good to know!